tests/jobserver/authorization/test_admins.py

```diff
@@ -6,6 +6,16 @@
 from ...factories import UserFactory
 
 
+@pytest.mark.django_db
+def test_ensure_admins_remove_user():
+    user = UserFactory(username="ghickman", roles=[SuperUser])
+
+    ensure_admins([])
+
+    user.refresh_from_db()
+    assert SuperUser not in user.roles
```

Comment:
Given importance of this function, could it be tested a bit more thoroughly?  For both the tests where `ensure_admins()` completes without error, there's only one user in the database, and you can imagine a bug in `ensure_admins()` so that it acts on all the existing users while still making the tests pass.

How about something like:

```
def test_ensure_admins():
    users = [
        UserFactory(username="aaa", roles=[SuperUser]),  # is superuser now, and will be afterwards
        UserFactory(username="bbb", roles=[SuperUser]),  # is superuser now, and won't be afterwards
        UserFactory(username="ccc"),  # isn't superuser now, and will be afterwards
        UserFactory(username="ddd"),  # isn't superuser now, and won't be afterwards
    ]

    ensure_admins(["aaa", "ccc"])

    assert SuperUser in User.objects.get(username="aaa").roles
    assert SuperUser not in User.objects.get(username="bbb").roles
    assert SuperUser in User.objects.get(username="ccc").roles
    assert SuperUser not in User.objects.get(username="ddd").roles
```

Reply:
Very good point, thanks 🙂  Fixed and rebased.